packages/modal/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@leafygreen-ui/modal",
+  "version": "0.9.0",
+  "description": "LeafyGreen UI Kit Modal",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "build": "../../node_modules/.bin/webpack --config ../../webpack.config.js", 
+    "ts:emit": "tsc --project ./tsconfig.json"
+  },
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@leafygreen-ui/lib": "^3.0.0",
+    "@leafygreen-ui/portal": "^1.1.0",
+    "@leafygreen-ui/palette": "^1.0.0",
+    "@leafygreen-ui/hooks": "^1.0.0",
+    "@leafygreen-ui/icon": "^1.0.4"
```

Comment:
Amazing to see our work starting to mesh. Is this the most leafygreen-ui components that we've used as dependencies to a component we've built?

Reply:
<3